src/System.Private.CoreLib/shared/System/Reflection/SignatureHasElementType.cs

```diff
@@ -36,8 +36,11 @@ protected SignatureHasElementType(SignatureType elementType)
         public sealed override Type[] GetGenericArguments() => Array.Empty<Type>();
         public sealed override Type[] GenericTypeArguments => Array.Empty<Type>();
         public sealed override int GenericParameterPosition => throw new InvalidOperationException(SR.Arg_NotGenericParameter);
-    
-        public sealed override string? Name => _elementType.Name + Suffix;
+
+#pragma warning disable CS8608
```

Comment:
this shouldn't be needed anymore, I think all of these are non-nullable now

Reply:
yes, updating